benchmark/jsonresultset/jsonresultset_test.go

```diff
@@ -133,3 +92,22 @@ func runJSONResultSet() {
 		counter++
 	}
 }
+
+func setCustomConfig() {
+	customJsonDecoderEnabled, err := sf.GetFromEnv("SNOWFLAKE_TEST_CUSTOME_JSON_DECODER_ENABLED", true)
+	if err != nil {
+		log.Fatal(err)
+	}
+	sf.CustomJSONDecoderEnabled = strings.EqualFold("true", customJsonDecoderEnabled)
+	maxChunkDownloadWorkers, err := sf.GetFromEnv("SNOWFLAKE_TEST_MAX_CHUNK_DOWNLOAD_WORKERS", false)
+	if err != nil {
+		log.Fatal(err)
+	}
+	if maxChunkDownloadWorkers != "" {
+		n0, err := strconv.Atoi(maxChunkDownloadWorkers)
```

Comment:
Boy scout rule ;) Maybe rename string variable with suffix `Str` and here name it `maxChunkDownloadWorkers` (same as goes for port).

Reply:
Fixed